README.md

```diff
@@ -166,6 +168,19 @@ and your webpack config is located at `/home/src/webpack.config.js`, then the va
 
 <br>
 
+#### EXCLUDE_RUNTIME
+
+`EXCLUDE_RUNTIME` is meant to be used with `render_entrypoint`. When creating multi-page applications, it's common to want to
+split a single runtime file that is used by all chunks. You can do that by using `{% render_bundle 'runtime' %}` in your base HTML file and setting `EXCLUDE_RUNTIME` to `True` in order to not include it again when using `{% render_entrypoint 'example_entry_point' %}`
+
+<br>
+
+#### BASE_ENTRYPOINT
+
+`BASE_ENTRYPOINT` is meant to be used with `render_entrypoint`. When creating multi-page applications, it's common to want to
+include common js in the base HTML. If the main entrypoint's name is `main`, you can do that by including `{% render_entrypoint 'main' %}` in your base HTML file. Now in another entrypoints (that extend the base HTML file), there might be some chunks that were already included in `main`, that means they would be included twice in the final rendered HTML, to avoid that, set `BASE_ENTRYPOINT` to `'main'`, then any duplicate chunks between an entrypoint and the main entrypoint would be included only once.
```

Comment:
Duplicate "When creating multi-page applications, it's common to want to
￼	include common js in the base HTML"

Reply:
That's not by mistake. If you have a clearer way to explain it, please suggest it to me, or do it yourself and push it here